### ISSUE TITLE
Exposure module bug with pipette

### ISSUE THREAD

**Issue by virtualritz** [darktable-org/darktable#2540]
Reported by Christian <chris66nrw@gmx.de> on darktable-dev:
OS=win7-64, DT=2.6.2 stable 

1) Activate the pipette-icon, Frame in the foto appears which may be
adusted by mouse. threashold slider is active now.

2) Change the Exposure-Slider

now the pipette-mode is turn-off/frame disappears, but the icon is
stay'ng highlighted.

Hit the pipette-icon again: frame is appearing/threashold slider is
active, but icon is grey (inactive) now.

In short: GUI-state of the pipette-icon is not reflecting the real
state.

**Comment by TurboGit** [comment]
This is fixed in master.
